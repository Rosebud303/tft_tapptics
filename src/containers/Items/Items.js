import React from 'react'

export default function Items() {
  return (
    <div>
      <h1>items, son</h1>
    </div>
  )
}
