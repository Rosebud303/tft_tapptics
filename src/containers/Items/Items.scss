.Items {
  background-image: linear-gradient($fade1, $fade2), url('../../assets/images/baron.jpg');
  background-size: cover;
  background-repeat: none;
  height: $page-height;
  padding: $page-padding;

  h3 {
    color: $body-font-color;
    font-size: 1.5rem;
    margin: 0;
  }

  .display-items {
    display: flex;
    flex-wrap: wrap;
    justify-content: space-around;
  }

  article {
    background-color: $modal-overlay;
    border: 1px solid $dark-bg;
    color: $btn-bg;
    display: grid;
    grid-template-areas: "img"
    "name"
    "info";
    grid-template-rows: 1fr .5fr .25fr;
    margin: auto;
    text-align: center;
    height: 12rem;
    width: 11%;

    h4, p, img {
      margin: .5rem;
    }

    img {
      grid-area: img;
      margin: auto;
    }

    h4 {
      grid-area: name;
    }

    p {
      font-size: .8rem;
      grid-area: info;
    }
  }
}