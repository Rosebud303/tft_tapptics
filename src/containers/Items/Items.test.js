import React from 'react';
import { shallow } from 'enzyme';
import Items from './Items';

describe('Items', () => {

  let wrapper;
  let mockItems = [{ attributes: { name: 'BF Sword'} }]

  beforeEach(() => {
    wrapper = shallow(<Items items={mockItems} />);
  });

  it('should match the snapshot with all the data passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });
});