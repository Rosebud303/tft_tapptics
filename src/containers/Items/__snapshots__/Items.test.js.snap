// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Items should match the snapshot with all the data passed in 1`] = `
<div>
  <h1>
    items, son
  </h1>
</div>
`;
